dialects/sqlite.yaml

```diff
@@ -37,8 +37,6 @@ scalar_functions:
   - name: modulus
     local_name: "%"
     infix: True
-    required_options:
```

Comment:
SQLite supports two different operations for modulus. 
1. `%` operator that will not for non-integer operands.
2. `mod` function that shall work for non-integer operands.

Reply:
Is it possible to test both?

Reply:
Not right now. AFAIK, right now the BFT assumes every dialect has only one implementation of a function. I will create an issue to make it change.